### ISSUE TITLE
Improve the workflow of copying one or multiple geomarks

### ISSUE THREAD

**Issue by gregseb** [pauperpythonistas/python-geomark#41]
We should try to improve the workflow of copying geomarks a little better.

When calling gm.copy() on an existing Geomark it works as expected.  Adding additional urls however... `gm.copy(geomarkUrl=['...', ...])` should append those additional urls to the one that was already used to create the `gm` Geomark object rather than overriding it.

Additionally there should be a way to simply call Geomark.copy([...]) with a list of GeomarkUrls or even perhaps with a list of Geomark objects without having to first create a Geomark object with one of the Geomark IDs.

**Comment by spatialbits** [comment]
Yep, part 1 of this definitely needs doing  and shouldn't require much work. I also suspect that the Geomark server is smart enough to strip off repeated IDs, so we could easily just stick the `self.id` in the list every time.

Part 2 - copy actually already works exactly as you're suggesting it should :D. It will take URLs or IDs. We could take it a little further and allow a list of Geomark objects and test for that in the copy method; it would deviate from the Geomark API though.
